sdks/python/apache_beam/examples/inference/sklearn_japanese_housing_regression.py

```diff
@@ -0,0 +1,164 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""A pipeline that uses RunInference API on a regression about housing prices.
+
+This example uses the japanese housing data from kaggle.
+https://www.kaggle.com/datasets/nishiodens/japan-real-estate-transaction-prices
+
+Since the data has missing fields, this example illustrates how to split
+data and assign it to the models that are trained on different subsets of
+features. The predictions are then recombined.
+
+In order to set this example up, you will need two things.
+1. Build models (or use ours) and reference those via the model directory.
```

Comment:
I am not seeing specific instructions in this PR, and no changes to sdks/python/apache_beam/examples/inference/README.md - what is the plan?

Reply:
Also, what is the plan for adding an IT for this example?

Reply:
We need to have a regression test for every example to catch any early bugs/regressions in the transform. 

Can you create one regression test with a smaller data and add that one to this (class)[https://github.com/apache/beam/blob/2c8e7eb7a39cbe3a1678a5c6b8b3f8700d4d8706/sdks/python/apache_beam/ml/inference/sklearn_inference_it_test.py#L41]

Reply:
I was planning to do that in a separate PR, but it's probably fine to do that here.